docs/source/terms_and_model.rst

```diff
@@ -1215,6 +1185,196 @@ order. See :ref:`computed-identifiers` for more information.
 
 
 
+.. _systemicvariation:
+
+SystemicVariation
+@@@@@@@@@@@@@@@@@
+
+
+AbsoluteAbundance
+#################
+
+**Biological Definition**
+
+AbsoluteAbundance is the absolute and quantified amount of an entity
+within a genome, cell, or sample.
+
+**Computational Definition**
+
+AbsoluteAbundance is represented as a `subject`, which may be an
+Allele or Haplotype, or any object identifiable with a CURIE.
+
+**Information Model**
+
+.. list-table::
+   :class: reece-wrap
+   :header-rows: 1
+   :align: left
+   :widths: auto
+
+   * - Field
+     - Type
+     - Limits
+     - Description
+   * - _id
+     - :ref:`CURIE`
+     - 0..1
+     - Computed Identifier
+   * - type
+     - string
+     - 1..1
+     - MUST be "AbsoluteAbundance"
+   * - subject
+     - :ref:`Allele` | :ref:`Haplotype` | :ref:`CURIE`
+     - 1..1
+     - Subject of the relative abundance statement
```

Comment:
change relative abundance to absolute abundance.

Reply:
Fixed